schemas/tls.yml

```diff
@@ -0,0 +1,174 @@
+  description: >
+    Fields related to TLS activity.
+  type: group
+  fields:
+
+    - name: version.protocol
+      level: extended
+      type: keyword
+      description: Normalized protocol name parsed from original string (e.g. ssl, tls).
+
+    - name: version.number
+      level: extended
+      type: keyword
+      description: Numeric part of the version parsed from the original string (e.g. 1.2, 3).
+
+    - name: cipher
+      type: keyword
+      level: extended
+      description: String indicating the cipher used during the current connection.
+
+    - name: curve
+      type: keyword
+      level: extended
+      description: String indicating the curve used for the given cipher, when applicable.
+
+    - name: resumed
+      type: boolean
+      level: extended
+      description: Boolean flag indicating if this TLS connection was resumed from an existing TLS negotiation.
+
+    - name: next_protocol
+      type: keyword
+      level: extended
+      description: String indicating the protocol being tunneled (e.g. http/1.1, spdy/3, imap, webrtc)
+
+    - name: client.ja3
+      type: keyword
+      level: extended
+      description: A hash that identifies clients based on how they perform an SSL/TLS handshake.
+
+    - name: client.server_name
+      type: keyword
+      level: extended
+      description: >
+        Also called an SNI, this tells the server which hostname to which the client is attempting to connect.
+
+    - name: client.supported_ciphers
+      type: keyword
+      level: extended
+      description: List of ciphers offered by the client during the client hello.
+
+    - name: client.subject
+      type: keyword
+      level: extended
+      description: Subject of the x.509 certificate presented by the client.
+
+    - name: client.issuer
+      type: keyword
+      level: extended
+      description: Subject of the issuer of the x.509 certificate presented by the client.
+
+    - name: client.not_before
+      type: date
+      level: extended
+      description: Timestamp indicating when client certificate is first considered valid.
+
+    - name: client.not_after
+      type: date
+      level: extended
+      description: Timestamp indicating when client certificate is no longer considered valid.
+
+    - name: client.certificate_chain
+      type: keyword
+      level: extended
+      description: List of PEM-encoded certificates that make up the certificate chain offered by the client.
+
+    - name: client.certificate
+      type: keyword
+      level: extended
+      description: PEM-encoded stand-alone certificate offered by the client.
+
+    - name: client.hash.md5
+      type: keyword
+      level: extended
+      description: >
+        Certificate fingerprint using the MD5 digest of DER-encoded version of certificate offered by the client.
+        For consistency with other hash values, this value should be formatted as an uppercase hash
+        (e.g. `0F76C7F2C55BFD7D8E8B8F4BFBF0C9EC`).
```

Comment:
I'd take out the parens and literally make it an example that gets picked up by the ECS scripts:

```YAML
      description: >
        Certificate fingerprint using the MD5 digest of DER-encoded version of certificate offered by the client.
        For consistency with other hash values, this value should be formatted as an uppercase hash.
      example: 0F76C7F2C55BFD7D8E8B8F4BFBF0C9EC
```

This will ensure it shows up as an example in the docs, but also in various other places such as the CSV